kinto/core/__init__.py

```diff
@@ -143,7 +143,10 @@ def __init__(self, fmt=None, datefmt=None, style="%"):
 
 def get_user_info(request):
     # Default user info (shown in hello view for example).
-    return {"id": request.prefixed_userid, "principals": request.prefixed_principals}
+    user_info = {"id": request.prefixed_userid, "principals": request.prefixed_principals}
+    if hasattr(request, "get_user_profile"):
```

Comment:
Maybe add a comment that this comes from the openid plugin

Reply:
I don't think it is only coming from the openid plugin, it is something any plugin can define.